examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  selector:
+    component: elasticsearch
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        image: quay.io/pires/docker-elasticsearch-kubernetes:1.7.1-4
```

Comment:
One thing I discussed with @brendandburns (for the Hazelcast example) and @satnam6502 for this example, was that we needed to test with images before pushing them to gcr.io/kubernetes. I have no issue with that, for as long as someone does it - I don't have perms for that.